eng/pipelines/azure-sdk-bom.yml

```diff
@@ -0,0 +1,20 @@
+trigger: none
+pr: none
+
+jobs:
+  - job: Build and Validate
+    pool:
+      vmImage: 'ubuntu-16.04'
+
+    steps:
```

Comment:
What about hooking up the signing and releasing parts of the pipeline? Do you intend to do that manually for now?

fyi @mitchdenny

Reply:
I'll want to add those, should have some follow-up changes to add this and make it generic for other potential BOMs we'll be releasing out of the repo.